tests/ssg_test_suite/virt.py

```diff
@@ -17,6 +17,8 @@ class SnapshotStack(object):
                      "     Full snapshot by SSG Test Suite"
                      "  </description>"
                      "</domainsnapshot>")
+    CREATE_FLAGS = libvirt.VIR_DOMAIN_SNAPSHOT_CREATE_ATOMIC
+    REVERT_FLAGS = self.REVERT_FLAGS
```

Comment:
what does this even do? you set a classmember using an instance member variable. but at the time you set it the instance doesn't exist

Reply:
Sorry, I though I am fast enough :)